test/saved/grad_rpsb_meoh_rhf_sto3g.out.ene

```diff
@@ -23,6 +23,6 @@
  ELECTRONIC ENERGY    = -2546.045274574
  CORE_CORE REPULSION  =  1688.269305715
  EXT CHARGE REPULSION =    -7.513193312
- TOTAL ENERGY         =  -857.775968859
+ TOTAL ENERGY         =  -957.775968859
```

Comment:
@vwcruzeiro This won't work. CIs will run successfully. In runtest script I am comparing .out ref files. Not .out.ene etc.

Reply:
Good catch. Let me try again.